docs/topics/intellij-idea-plugin-run-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-run-configuration_{context}"]
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project you import to IntelliJ IDEA.
+
+.Procedure
+
+. In *Projects* view, click the project you want to analyze.
+. On the left side of the screen, click the *{ProductName}* tab.
++
+If this is your first configuration, the run configuration panel is displayed on the right.
+
+. If this is not your first configuration, right-click on one of the configurations in the list and select *New configuration*.
++
+The run configuration panel is displayed on the right.
+
+. Complete the following configuration fields:
+
+** *mta-cli*: Click *Add*, select the filepath, and click *Save*.
+** *Input*: Click *Add* and do one of the following:
+
+*** Enter the input file or directory and press Enter.
+*** Click *Open File Explorer*, select the directory, and click *OK*.
+
+** *Target*: Select one or more target migration paths.
++
+[NOTE]
+====
+The location shown in the *Output* is set by the plugin.
+====
+
+. In the list of configurations, right-click on the new configuration and select *Run Analysis*.
++
+The *Console (MTA)* terminal emulator opens, displaying information about the progress of the analysis.
++
+When the analysis is completed, you can click either *Report* or *Results* below the name of the configuration file you ran.
+
+** *Reports* opens the MTA report, which describes any issues you need to address before you migrate or modernize your application. For more information, see link:{ProductDocUserGuideURL}#review-reports_cli-guide[MTA report] in the _{UserCLIBookName}_.
+
+** *Results* opens a directory displaying Hints (issues) per application.
```

Comment:
Why is Hints capitalized here?

Reply:
You're right. I'm not using it as a proper noun or a GUI element (and there's no need to). Changed.